src/main/java/com/bitpay/sdk/model/Payout/PayoutInstruction.java

```diff
@@ -164,4 +165,15 @@ public String getStatus() {
     public void setStatus(String status) {
         this._status = status;
     }
+
+    @JsonProperty("address")
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+    public String getAddress() {
+        return _address;
+    }
+
+    @JsonProperty("address")
+    public void setAddress(String email) {
```

Comment:
replace "email" for address and make sure you can create an instance of each one of the modified models successfully before submitting changes. Use the Create methods to test it.